source/Utility/ReproducerInstrumentation.cpp

```diff
@@ -119,13 +119,4 @@ Recorder::~Recorder() {
   UpdateBoundary();
 }
 
-void Recorder::Log(unsigned id) {
```

Comment:
How did this compile in the first place? Is there a matching declaration in a header that should be removed too?

Reply:
Yes. There's a matching call in the header that needs to be removed.

Reply:
Yeah we should update the PR to remove that as well.